superchic.sh

```diff
@@ -0,0 +1,59 @@
+package: SuperCHIC
+version: "%(tag_basename)s%(defaults_upper)s"
+tag: "alice/v2.04"
+source: https://github.com/hcab14/SuperCHIC.git
+requires:
+ - lhapdf
+build_requires:
+ - curl
+ - "GCC-Toolchain:(?!osx)"
+---
+#!/bin/bash -e
+
+rsync -a --exclude '**/.git' $SOURCEDIR/ ./
+
+patch makefile <<EOF
```

Comment:
What is this? This should be fixed upstream, and the repository is under our control. We have banned in-recipe parsing since when we've started using aliBuild :-)